src/ubx.cpp

```diff
@@ -1823,6 +1884,12 @@ GPSDriverUBX::payloadRxAddMonVer(const uint8_t b)
 
 			if (fwver_str != nullptr) {
 				GPS_INFO("u-blox firmware version: %s", fwver_str + strlen("FWVER="));
+
+				// Check if its a ZED-F9P-15B
+				if (strstr(fwver_str, "HPGL1L5") && (_board == Board::u_blox9)) {
```

Comment:
Nitpick, check the easy one first.

```suggestion
				if ((_board == Board::u_blox9) && strstr(fwver_str, "HPGL1L5")) {
```

Reply:
Done